src/MolSim.cpp

```diff
@@ -1,50 +1,136 @@
 
 #include "FileReader.h"
 #include "outputWriter/XYZWriter.h"
+#include "outputWriter/VTKWriter.h"
 #include "utils/ArrayUtils.h"
 
 #include <iostream>
 #include <list>
 
+#include  <getopt.h>
```

Comment:
Overall, it is a fine decision for the practical course 👍(So, I wouldn't request any change), just as a heads-up:

This is not a C++-Standard-Header, but rather a POSIX header. Hence, it is only available on POSIX systems (i.e. Linux, macOS), but not e.g. on Windows.

If you develop in a professional context, you might consider platform-independent portable solutions (e.g., [argparse](https://github.com/p-ranav/argparse)).

This is going beyond the assignment and no request-to-change, but you are here for learning. So, I just wanted to point it out :)